bigquery/deployment/src/main/java/io/quarkiverse/googlecloudservices/bigquery/deployment/BigQueryBuildSteps.java

```diff
@@ -17,4 +20,12 @@ public FeatureBuildItem feature() {
     public AdditionalBeanBuildItem producer() {
         return new AdditionalBeanBuildItem(BigQueryProducer.class);
     }
+
+    @BuildStep
+    public List<RuntimeInitializedClassBuildItem> runtimeInitializedClass() {
```

Comment:
Thanks you exploring this further @loicmathieu! Could you be provide more information on the error you received without these configurations? We recently had to add a few configurations for [Apache Arrow in java-bigquery](https://github.com/googleapis/java-bigquery/blob/c5c97b0c6fd98978e3e480cd565bf4f832b35e7c/google-cloud-bigquery/src/main/resources/META-INF/native-image/com.google.cloud/google-cloud-bigquery/reflect-config.json#L3) but did not run into an initialization issue so I'm wondering if there a special scenario that we might have missed.